addons/armaos/functions/fnc_shell_validatePassword.sqf

```diff
@@ -0,0 +1,41 @@
+/**
+ * Tries to validate given username and password. If successful switches to shell mode.
+ * If not successful switches back to login mode.
+ *
+ * Arguments:
+ * 1: Computer <OBJECT>
+ *
+ * Results:
+ * None
+ */
+
+params ["_computer"];
+
+private _terminal = _computer getVariable "AE3_terminal";
+
+private _username = _terminal get "AE3_terminalLoginUser";
+private _password = _terminal get "AE3_terminalPasswordBuffer";
+
+private _users = _computer getVariable [ "AE3_Userlist", [["topsykrett", "test123"]] ];
```

Comment:
Why do we need a default user here?
Empty list would be sufficient.

Reply:
removed in commit https://github.com/y0014984/Advanced-Equipment/pull/90/commits/351ca2f7548ca9717ff26f606317f2c7e56c0763